server/config.ts

```diff
@@ -103,5 +103,6 @@ export default {
     audit: auditConfig(),
   },
   ingressUrl: get('INGRESS_URL', 'http://localhost:3000', requiredInProduction),
+  dpsUrl: get('DPS_URL', 'http://localhost:8281', requiredInProduction),
```

Comment:
also realised that I need the `helm_deploy` changes to add in the `DPS_URL` value for all environments